source/AMDSCentralServer.cpp

```diff
@@ -133,3 +108,36 @@ void AMDSCentralServer::onHundredMillisecondTimerUpdate(){
 
 	amptek1BufferGroup_->append(oneSpectralDataHolder);
 }
+
+void AMDSCentralServer::initializeBufferGroup(quint64 maxCountSize)
+{
+	QList<AMDSAxisInfo> mcpBufferGroupAxes;
+	mcpBufferGroupAxes << AMDSAxisInfo("X", 1024, "X Axis", "pixel");
+	mcpBufferGroupAxes << AMDSAxisInfo("Y", 512, "Y Axis", "pixel");
```

Comment:
Just for my own sake, what is the Y axis you refer to here?  The 1024 is the channel bins, but I don't see what the Y axis is.